Src/Support/Google.Apis.Auth/SignedTokenVerification.cs

```diff
@@ -243,9 +244,14 @@ public async Task<IEnumerable<AsymmetricAlgorithm>> GetCertificatesAsync(string
                         cachedCerts.Expired(_clock.UtcNow))
                     {
                         string certificatesJson = await FetchCertificatesAsync(certificatesLocation).ConfigureAwait(false);
+                        IEnumerable<JToken> jwks = JToken.Parse(certificatesJson)["keys"]?.AsEnumerable() 
+                            ?? throw new ArgumentException($"Only JWK formatted keys are currently supported. No 'keys' element was found in {certificatesLocation}");
+                        if (!jwks.Any())
+                        {
+                            throw new ArgumentException($"No JWKs were found on {certificatesLocation}. The 'keys' element was empty.");
+                        }
                         cachedCerts = new CachedCertificates(
-                            JToken.Parse(certificatesJson)["keys"]
-                            .AsEnumerable()
+                            jwks
```

Comment:
Unwrap the line below into this one?

Reply:
Yes, done, everything fit neetly in one line now. I'll push again and merge on green. Thanks!